lib/pytap13.py

```diff
@@ -17,43 +17,37 @@
 # Author: Josef Skladanka <jskladan@redhat.com>
 
 import re
-
-try:
-    # Python 2
-    from StringIO import StringIO
-except ImportError:
-    # Python 3
-    from io import StringIO
+from io import BytesIO
 
 import yaml
-from lib.utils import string_types
 
+from lib.utils import string_types, binary_regex, str_to_bytes, bytes_to_str
 
-RE_VERSION = re.compile(r"^\s*TAP version 13\s*$")
-RE_PLAN = re.compile(
+RE_VERSION = binary_regex(r"^\s*TAP version 13\s*$")
```

Comment:
Why not just `re.compile(b."....")`?

Reply:
Binary strings with the prefix "b" are allowed only in python3. We discussed this approach with additional function with @Totktonada

Reply:
There is "br" literal in python 2.7. Fixed.